lb_matching_tools/guff.py

```diff
@@ -0,0 +1,80 @@
+GUFF_PAREN_WORDS = [
+    "a_cappella",
```

Comment:
If I remember correctly, guess case joins the two words with an underscore temporarily because it can only handle a single word.
So you probably have to replace underscores with spaces, maybe it's even worth adding the common (mis)spelling as a single word.